.devcontainer/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# /********************************************************************************
+# * Copyright (c) 2022 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+# Download base image of Ubuntu
+FROM ghcr.io/boschglobal/kuksa.val.services/oci_kuksa-val-services-ci:prerelease
```

Comment:
@SebastianSchildt How shall we handle repo changes?

Reply:
"as we like", 

as end-goal, we probably would want to have 

```
ghcr.io/eclipse/kuksa.val.services/oci_kuksa-val-services-ci:prerelease
```

For that we (I) would need to request a Github Actions secret from Eclipse so that the Actions in that repo can also push an image

For testing/if it is easier, wen could even leave the path pointing to boshglobal in the beginning, if we have pushed an image here. As boschglobal is open, it should also work for users. Then we can make another PR in upstream once we migrated.